zebra/zebra_routemap.c

```diff
@@ -397,7 +417,7 @@ DEFUN (no_set_src,
 
 ALIAS (no_set_src,
        no_set_src_val_cmd,
-       "no set src (A.B.C.D)",
+       "no set src (A.B.C.D|X:X::X:X)",
```

Comment:
It's a special parameter inside of quagga. Check https://github.com/Azure/sonic-quagga/blob/debian/0.99.24.1/lib/command.h#L264